src/aiod/__init__.py

```diff
@@ -17,5 +16,3 @@
 from aiod.resources import publications
 from aiod.resources import services
 from aiod.resources import teams
```

Comment:
Regarding the `from aiod import config`. Why not simply import it here in the `__init__.py` file?
```suggestion
from aiod.resources import teams
from aiod.config._config import config
```

Reply:
A third thing we could do is for the `calls/urls.py` and others to not import it from `aiod` but instead directly from `aiod.config._config`. Might be better.

Reply:
Great, I'll push your suggestions in a minute with a tiny modification:
1. Rename `config` module to `configuration`
2. Include `from _config import config` at `configuration/__init__.py`
3. Include `from aiod.configuration import config` at `aiod.__init__.py`
4. Import `config` from `aiod` directly from `aiod.configuration` in the files such as `calls/urls.py`

Looks cleaner to me.

Reply:
Agreed. Great!